editor/src/components/inspector/sections/image-section/image-source-control.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { treeToContents } from '../../../../components/assets'
+import { isImageFile } from '../../../../core/model/project-file-utils'
+import { ProjectContents } from '../../../../core/shared/project-file-types'
+import { useEditorState } from '../../../editor/store/store-hook'
+import { useInspectorElementInfo } from '../../common/property-path-hooks'
+import { OptionChainControl } from '../../controls/option-chain-control'
+import { SelectControl, SelectOption } from '../../controls/select-control'
+import { StringControl } from '../../controls/string-control'
+
+enum ImageSrcType {
```

Comment:
nit: I think string unions are more idiomatic in typescript (since enums don't exist in JS)

Reply:
the autocomplete lied to me there, probably because of the lookup I did earlier on another usage. fixed here https://github.com/concrete-utopia/utopia/pull/2660/commits/2ecf83390df716fe8886464596bf2e7bad3c2a5a if that looks ok :)